Dev/VideoChatClient/Main.cpp

```diff
@@ -2,13 +2,13 @@
 
 
 #include "AutoInitWinSock/AutoInitWinSock.hpp"
+#include "Input/ImageInput.hpp"
 #include "Socket/Socket.hpp"
 
-
-#include "opencv2/opencv.hpp"
-
 int wmain()
 {
+	ImageInput im{};
```

Comment:
We should think how we want to use the input/output generically, that we can send image and after that voice while keeping the right expected flow

Reply:
Thought about it, maybe double Inheritance? Or another class for AV